File: Arcade/TheCore/Java/FirstReverseTry.java
final class FirstReverseTry {
	
	int[] firstReverseTry(int[] arr) {
	    if (arr.length > 0) {
	        int tmp = arr[0];
	        arr[0] = arr[arr.length - 1];
	        arr[arr.length - 1] = tmp;
	    }
	    
	    return arr;
	}
}

File: Arcade/TheCore/Kotlin/firstReverseTry.kt
fun firstReverseTry(arr: MutableList<Int>): MutableList<Int> {
    fun <T> MutableList<T>.swp(i: Int, j: Int): Unit {
        val tmp: T = this[i];
        this[i] = this[j];
        this[j] = tmp;
    };
    
    if (arr.isNotEmpty()) arr.swp(0, arr.size - 1)
    
    return arr;
}